lighthouse-core/audits/time-to-interactive.js

```diff
@@ -27,8 +27,8 @@ class TTIMetric extends Audit {
     return {
       category: 'Performance',
       name: 'time-to-interactive',
-      description: 'Time To Interactive (alpha)',
```

Comment:
oh, right. Yeah, it's `load-fast-enough-for-pwa` that needs updating to First Interactive